http/src/v0.rs

```diff
@@ -185,7 +185,7 @@ mod tests {
         routes(&ipfs, shutdown_tx)
     }
 
-    #[tokio::test(max_threads = 1)]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
```

Comment:
nit: the default test runtime is now single-threaded, so these could potentially just be `#[tokio::test]` now

Reply:
Aah I had missed that great!